tests/storage/hotplug.go

```diff
@@ -1330,7 +1330,7 @@ var _ = SIGDescribe("Hotplug", func() {
 
 		AfterEach(func() {
 			if lr != nil {
-				err = virtClient.CoreV1().LimitRanges(namespace).Delete(context.Background(), lr.Name, metav1.DeleteOptions{})
+				err = virtClient.CoreV1().LimitRanges(testsuite.GetTestNamespace(nil)).Delete(context.Background(), lr.Name, metav1.DeleteOptions{})
```

Comment:
For consistency I'd prefer to use `lr.Namespace`
```suggestion
				err = virtClient.CoreV1().LimitRanges(lr.Namespace).Delete(context.Background(), lr.Name, metav1.DeleteOptions{})
```

Reply:
Also, this could be a great opportunity to rename:
- `namespace` -> `namespaceArg`
- `size` -> `sizeArg`
- `insecure` - > `insecureArg`
- `imageUpload` -> `imageUploadCommand`
These are just suggestions, not mandatory things.
Thanks

Reply:
I would go with the lr.Namespace suggestion. In older versions `testsuite.GetTestNamespae` doesn't exist